src/plugins/analysis/strings/internal/string_eval.py

```diff
@@ -0,0 +1,132 @@
+"""
+evaluate relevance of strings
+
+Credits:
+Original version by Paul Schiffer created during Firmware Bootcamp WT16/17 at University of Bonn
+Refactored and improved by Fraunhofer FKIE
+"""
+
+from __future__ import annotations
+
+import re
+
+REGEX_LOWER = re.compile(r'[a-z]')
+REGEX_CAPITAL = re.compile(r'[A-Z]')
+REGEX_WORD = re.compile(r'[a-zA-Z]')
+REGEX_NO_WORD = re.compile(r'\W')
+QUAD_CHAR_REGEX = re.compile(r'(\S)\1\1\1')
+YEAR_REGEX = re.compile(r'(19\d\d)|(20\d\d)')
+PATH_REGEX = re.compile(r'(/[\w-]+)+(.[a-zA-Z]+)')
+VERSION_REGEX = re.compile(r'\d+\.(\d+\.?)+')
+FORMAT_STR_REGEX = re.compile(r'%s|%lu|%u|%lf|%f|%i|%d')
+MAIL_REGEX = re.compile(
+    r'(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))'
+    r'@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))'
+)
+PREFIX_REGEX = re.compile(r'(^_+)|^\.')
+PARAMETER_REGEX = re.compile(r'^\s*-{1,2}')
+HTML_REGEX = re.compile(r'</?[^\\()$\[\]§.,?<>;|!]+>')
+
+DICTIONARY = {'version', 'v.', 'http', 'ftp', 'usage', 'Usage', 'ssh', 'SSH', 'password', 'Version'}
+
+
+def evaluate_string(string: str) -> float:
```

Comment:
nit: What about renaming this to `string_relevance`?

Reply:
`string_relevance` is lacking a verb. I renamed it to `calculate_relevance_score`